tasks/winbuildscripts/integrationtests.ps1

```diff
@@ -5,21 +5,16 @@ New-LocalUser -Name "ddagentuser" -Description "Test user for the secrets featur
 $Env:Python2_ROOT_DIR=$Env:TEST_EMBEDDED_PY2
 $Env:Python3_ROOT_DIR=$Env:TEST_EMBEDDED_PY3
 
-if ($Env:TARGET_ARCH -eq "x64") {
-    & ridk enable
-}
+& ridk enable
 & $Env:Python3_ROOT_DIR\python.exe -m  pip install -r requirements.txt
 
 $UT_BUILD_ROOT=(Get-Location).Path
 $Env:PATH="$UT_BUILD_ROOT\dev\lib;$Env:GOPATH\bin;$Env:Python3_ROOT_DIR;$Env:Python3_ROOT_DIR\Scripts;$Env:PATH"
 
-$archflag = "x64"
-if ($Env:TARGET_ARCH -eq "x86") {
-    $archflag = "x86"
-}
+& $Env:Python3_ROOT_DIR\python.exe -m pip install PyYAML==5.3.1
```

Comment:
```suggestion
```

Reply:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice catch, thanks ! Fixed in [e89bb3d](https://github.com/DataDog/datadog-agent/pull/21240/commits/e89bb3ddb5b47a582f065b6114e7863ea459f227).